index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+
+  <body>
+    <p id="para-1">Hello World we are software developers!</p>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <title id="title">Donald Trump</title>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://i.guim.co.uk/img/media/70d12c1d77662696f75a21d04ec1bb5e8a702390/60_37_4451_2672/master/4451.jpg?width=465&quality=45&auto=format&fit=max&dpr=2&s=209e10f34772bc8b4e9a58fa8227dd5a"
+          alt="Donald Trump waving to crowd"
+          class="responsive"
+        />
```

Comment:
It is a good idea to give descriptive classes to our elements to make our code more explanatory and make it easier to navigate through our code.  For example, we could use class="donald-image" or "main-image". To make our code more accessible.